bin/common.mli

```diff
@@ -58,7 +58,7 @@ val init :
   -> t
   -> Dune_config.t
 
-(** [examples \[("description", "dune cmd foo"); ...\]] is an [EXAMPLES] manpage
+(** [examples [("description", "dune cmd foo"); ...]] is an [EXAMPLES] manpage
```

Comment:
I think we meant this to be a list. I mean, the output should include a literal `[`. Is this what this does?

Reply:
Yes, balanced `[]` don't need escaping.